### DISCUSSION TITLE
Add new Option to Already Created Product Variant

### DISCUSSION THREAD

**Discussion by dipit2007** [lunarphp/lunar#1075] [Feature Requests]
Add new option is not available for already created Product variant.

**Comment by glennjacobs** [comment]
@wychoong was this something your PR #798 was looking to address?

**Reply by wychoong** [reply]
it does solve this issue but its not the main goal of it

**Reply by dipit2007** [reply]
@glennjacobs I think no .. I am talking about product variant add/edit with add new option.

**Reply by glennjacobs** [reply]
Yes, so am I. We need to work with @wychoong to finish up that development which we'll be doing soon.

**Reply by wychoong** [reply]
![image](https://github.com/lunarphp/lunar/assets/67364036/13abc84e-952f-477d-978e-0be9000613c6)

in that PR, the `Select option` button allows you to add new Option
in the section there is a trash button to remove an Option
- and the tick/untick to add/remove OptionValue

which all these changes can add/remove a product variant

in it's current form you can't change a variant from `color:Red` -> `color:Blue`

hope this explanation helps

**Reply by dipit2007** [reply]
@wychoong where is this form  available? Add product is OK. But after adding product i want to add new option like "Shoe Lace".

**Reply by wychoong** [reply]
> @wychoong where is this form available? Add product is OK. But after adding product i want to add new option like "Shoe Lace".

its in the PR I'm proposing Glenn mentioned

**Reply by dipit2007** [reply]
OK Thank you.

![image](https://github.com/lunarphp/lunar/assets/7821088/30dcff45-f537-4489-8302-aaabd31cbf36)

In above pic there are two options with values. And I need to add third option to each variant. Hope your PR will help me. Thanks a lot.

**Comment by dipit2007** [comment]
@glennjacobs when it will be released?

**Reply by dipit2007** [reply]
@glennjacobs when it will be released?
